install/0000_30_machine-api-operator_07_machinehealthcheck.crd.yaml

```
@@ -50,6 +50,10 @@ spec:
           spec:
             description: Specification of machine health check policy
             properties:
+              maxAge:
+                description: MaxAge maximal age of a machine before it gets remediated Expects an unsigned duration string of decimal numbers each with optional fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

looks like a period is missing
```suggestion
                description: MaxAge maximal age of a machine before it gets remediated. Expects an unsigned duration string of decimal numbers each with optional fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```